sonar-ws/src/main/protobuf/ws-organizations.proto

```diff
@@ -38,7 +38,8 @@ message UpdateWsResponse {
 
 // WS api/organizations/search
 message SearchWsResponse {
-  repeated Organization organizations = 1;
+  optional sonarqube.ws.commons.Paging paging = 1;
+  repeated Organization organizations = 2;
```

Comment:
Indeed good point, I've forgotten that this WS were already used by the Scanner, I will change it